content/en/continuous_integration/setup_pipelines/gitlab.md

```diff
@@ -149,6 +149,48 @@ through the UI by going to **Settings > CI/CD > Runners** and editing the approp
 After these steps, CI Visibility adds the hostname to each job. To see the metrics, click on a job span in the trace
 view. In the drawer, a new tab named **Infrastructure** appears which contains the host metrics.
 
+### Error messages for pipeline failures
+
+For failed GitLab pipeline executions (on the [Pipeline Executions][5] page), each error under the `Errors` tab within a specific execution displays a message associated with the error type from GitLab.
```

Comment:
"For failed GitLab pipeline executions (on the Pipeline Executions page)" sounds like you can only see this from the Pipeline Executions page you've linked to.

Can you re-word to make it clear that users will always be able to see this whenever they go into any pipeline execution and that the page linked is one of which places you can see it?

Reply:
Removed the link as I think it made things more confusing. Thanks!